### PR TITLE
Seed data generation

### PR DESCRIPTION
**Pull request by yiyang381**
Created seed file and validation for the house model, please review!

### CODE REVIEW

File: app/models/house.rb [hunk 1]
```diff
@@ -1,4 +1,12 @@
 class House < ApplicationRecord
+  TENURE_CAT = ["99-year", "999-year", "freehold"]
+  BEDROOM = [1, 2, 3, 4, 5]
+  BATHROOM = [1, 2, 3, 4, 5]
+  PROPERTY_TYPE = ["HDB", "Condo", "Landed"]
+
   belongs_to :user
+  validates :address, :price, :bedroom, :bathroom, :square_feet, :description, :property_type, :tenure, presence: true
+  validates :bedroom, :bathroom, numericality: { greater_than_or_equal_to: 1, less_than_or_equal_to: 5 }
```
Comment (douglashjf) [inline app/models/house.rb:9]: can we make price and square feet validation for integers greater than 0?

**Review by douglashjf — CHANGES_REQUESTED** [review]
Hi YY, can you adjust the validation in the model

File: app/models/house.rb (hunk 1 above)
Comment (yiyang381) [reply app/models/house.rb:9]: done

**Comment by douglashjf** [comment]
superb

**Pull request merged**